### PR TITLE
LOOM-1306 BpkPaginationNudger Class 2 Overrides

### PR DESCRIPTION
**Pull request by jronald01**
**Class 2 Overrides**:

- Removed redundant left/right arrow styling
- Moved button styling to wrapper
  - Had to reduce `padding-top` to * by 2.75 rather than 10 to get visual test and visual test with zoom to match

Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated
- [ ] Type declaration (`.d.ts`) files updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: packages/bpk-component-pagination/src/BpkPaginationNudger.js [hunk 1]
```diff
@@ -31,37 +31,24 @@ const AlignedArrowLeftIcon = withRtlSupport(ArrowLeftIcon);
 const AlignedArrowRightIcon = withRtlSupport(ArrowRightIcon);
 
 const nudgerIcon = (forward) =>
-  forward ? (
-    <AlignedArrowRightIcon
-      // TODO: className to be removed
-      // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-      className={getClassName('bpk-pagination-nudger__icon')}
```
Comment (Sybsw) [inline packages/bpk-component-pagination/src/BpkPaginationNudger.js:38]: Where was this `__icon` referring to?

**Review by Sybsw — APPROVED** [review]

File: packages/bpk-component-pagination/src/BpkPaginationNudger.module.scss [hunk 2]
```diff
@@ -23,7 +23,10 @@
   display: flex;
   width: tokens.bpk-spacing-xl() + tokens.bpk-spacing-sm();
   height: tokens.bpk-spacing-xl() + tokens.bpk-spacing-sm();
-  padding: tokens.$bpk-one-pixel-rem * 10;
+  padding-top: tokens.$bpk-one-pixel-rem * 2.75;
+  padding-right: tokens.$bpk-one-pixel-rem * 10;
+  padding-bottom: tokens.$bpk-one-pixel-rem * 10;
+  padding-left: tokens.$bpk-one-pixel-rem * 10;
```
Comment (mungodewar) [inline packages/bpk-component-pagination/src/BpkPaginationNudger.module.scss:29]: I think we can get rid of these, just writing up a demo

Comment (jronald01) [reply packages/bpk-component-pagination/src/BpkPaginationNudger.module.scss:29]: Oops, have merged it - I'll create another PR once I can see what you are suggesting Mungo

Comment (mungodewar) [reply packages/bpk-component-pagination/src/BpkPaginationNudger.module.scss:29]: Yeah, no worries. I'll fire it across and see what you think.

Comment (jronald01) [reply packages/bpk-component-pagination/src/BpkPaginationNudger.module.scss:29]: It also works with:
```
    padding-top: tokens.$bpk-one-pixel-rem * 2.75;
    justify-content: space-evenly;
```

**Pull request merged**